tesseract_motion_planners/include/tesseract_motion_planners/descartes/problem_generators/default_problem_generator.h

```diff
@@ -70,6 +70,10 @@ DefaultDescartesProblemGenerator(const std::string& name,
     CONSOLE_BRIDGE_logError("No Inverse Kinematics solver found");
     return prob;
   }
+
+  // Synchronize the inverse kinematics with the forward kinematics
+  prob->manip_inv_kin.sychronize(prob->manip_fwd_kin);
```

Comment:
```suggestion
  prob->manip_inv_kin->synchronize(prob->manip_fwd_kin);
```
But it still gives an error
```
error: passing ‘std::__shared_ptr_access<const tesseract_kinematics::InverseKinematics, (__gnu_cxx::_Lock_policy)2, false, false>::element_type’ {aka ‘const tesseract_kinematics::InverseKinematics’} as ‘this’ argument discards qualifiers [-fpermissive]
   prob->manip_inv_kin->synchronize(prob->manip_fwd_kin);
```